### PR TITLE
chore(tests): fix up tests for 1.4.6 integration

### PR DESCRIPTION
**Pull request by matsko**

### CODE REVIEW

File: test/angular-material-mocks.js [hunk 1]
```diff
@@ -32,6 +32,26 @@ angular.module('ngMaterial-mock', [
...
+      if (asyncFlush) {
+        asyncRun = function() {
+          asyncFlush.flush();
+        };
+      }
+      
+      $delegate.flush = function() {
+        $rootScope.$digest();
+        $$rAF.flush();
+        $timeout.flush();
```
Comment (ThomasBurleson) [inline test/angular-material-mocks.js]: $timeout calls a $digest() internall

Comment (matsko) [inline test/angular-material-mocks.js]: You still need timeout because of `$mdUtil.nextTick`.

**Comment by jelbourn** [comment]
@ThomasBurleson

**Comment by jelbourn** [comment]
General LGTM (not going to bother with style comments here).

**Pull request closed without merging**